roles/keystone-defaults/defaults/main.yml

```diff
@@ -15,6 +15,7 @@ keystone:
       distro: /usr/lib/python2.7/site-packages/keystone
       source: /opt/openstack/current/keystone
       package: /opt/openstack/current/keystone
+      buffer_size: 114688
```

Comment:
This needs to be indented2 left for it to be read as keystone.uwsgi.buffer_size
Better to move this variable right under uwsgi @line 10